### PR TITLE
DRILL-4499: Remove 17 unused classes

### PR DESCRIPTION
**Pull request by sudheeshkatkam**

### CODE REVIEW

**Comment by sudheeshkatkam** [comment]
@adeneche please review. Easiest of all :)

**Comment by adeneche** [comment]
+1, LGTM

File: exec/interpreter/src/test/java/org/apache/drill/exec/expr/TestPrune.java [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
- * See the License for the specific language governing permissions and
- * limitations under the License.
- ******************************************************************************/
-package org.apache.drill.exec.expr;
-
-import org.apache.drill.BaseTestQuery;
-import org.apache.drill.common.util.TestTools;
-import org.junit.Test;
-
-public class TestPrune extends BaseTestQuery {
```
Comment (adeneche) [inline exec/interpreter/src/test/java/org/apache/drill/exec/expr/TestPrune.java]: why did you remove this unit test ?

Comment (sudheeshkatkam) [inline exec/interpreter/src/test/java/org/apache/drill/exec/expr/TestPrune.java]: I do not exactly remember what @amansinha100 mentioned.

I'll remove this change from my PR; and maybe the test can be moved to another directory, rather than removed.
